### ISSUE TITLE
Why are there more HEALTH AND HUMAN SERVICES from nysdoh_healthfacilities?

### ISSUE THREAD

**Issue by AmandaDoyle** [NYCPlanning/db-facilities#454]
Why are there more HEALTH AND HUMAN SERVICES from nysdoh_healthfacilities? New data on HOME CARE SERVICES AGENCY. Look at why we went from 0 to 700.

**Comment by mgraber** [comment]
@AmandaDoyle  is this addressed by #453 or a different issue?

**Comment by SPTKL** [comment]
I think so, factype for nysdoh_healthfacilities is taken directly from the description field from the raw data, the fact that it went from 0 -> 700 means that it's a new category in the raw data. 
```sql
select * from nysdoh_healthfacilities."2020/10/27" where description ~* 'HOME CARE SERVICES AGENCY'
```
gives 0 records
